standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/MetaStoreFilterHook.java

```diff
@@ -85,15 +85,13 @@ default List<String> filterCatalogs(List<String> catalogs) throws MetaException
   List<String> filterTableNames(String catName, String dbName, List<String> tableList)
       throws MetaException;
 
-  // Previously this was handled by filterTableNames.  But it can't be anymore because we can no
-  // longer depend on a 1-1 mapping between table name and entry in the list.
   /**
    * Filter a list of TableMeta objects.
    * @param tableMetas list of TableMetas to filter
    * @return filtered table metas
    * @throws MetaException something went wrong
    */
-  List<TableMeta> filterTableMetas(String catName,String dbName,List<TableMeta> tableMetas) throws MetaException;
+  List<TableMeta> filterTableMetas(List<TableMeta> tableMetas) throws MetaException;
```

Comment:
I removed catName and dbName since TableMeta object has this information. I do believe this interface is Hive internal (or the only users is the Hive code base).

Reply:
nit: I'm not sure the right way, perhaps we can mark the original method DEPRECATED and print WARN message

Reply:
I'll make this change. I had hoped to just drop this interface method since I didn't see any indication of other systems implementing MetaStoreFilterHooks (I generally think less dead code laying around the better), but I guess it is better to be safe here.